proxy/fluentd/README.md

```diff
@@ -0,0 +1,133 @@
+
+## Fluentd features
+
+Some interesting Fluentd features are available according to the [official documentation](https://docs.fluentd.org/):
+
+* [Buffering/retrying parameters](https://docs.fluentd.org/output#buffering-retrying-parameters),
+* Useful output plugins:
+   * [Output Kafka plugin](https://docs.fluentd.org/output/kafka),
+   * [Output S3 plugin](https://docs.fluentd.org/output/s3),
+   * [Output copy plugin](https://docs.fluentd.org/output/copy),
+   * [Output HTTP plugin](https://docs.fluentd.org/output/http) with options such as [retryable_response_codes](https://docs.fluentd.org/output/http#retryable_response_codes) to specify backend codes that should cause a retry,
+* [Buffer configuration](https://docs.fluentd.org/configuration/buffer-section),
+* [Embedding Ruby Expressions in config files to contain environment variables](https://docs.fluentd.org/configuration/config-file#embedding-ruby-expressions).
+
+The official Fluentd documentation does not mention guarantees about event ordering. However, retrieving
+Openlineage events and buffering in file/memory should be considered a millisecond-long operation,
+while any HTTP backend cannot guarantee ordering in such a case. On the other hand, by default
+the amount of threads to flush the buffer is set to 1 and configurable ([flush_thread_count](https://docs.fluentd.org/output#flush_thread_count)).
+
+## Quickstart with Docker
+
+Please refer to the [`Dockerfile`](docker/Dockerfile) and [`fluent.conf`](docker/conf/fluent.conf) to see how to build and install the plugin with
+the example usage scenario provided in [`docker-compose.yml`](docker/docker-compose.yml). To run the example setup, go to the `docker` directory and execute the following command:
+
+```shell
+docker-compose up
+```
+
+After all the containers have started, send some HTTP requests:
+
+```shell
+curl -X POST \
+-d '{"test":"test"}' \
+-H 'Content-Type: application/json' \
+http://localhost:9880/api/v1/lineage
+```
+In response, you should see the following message:
+
+`Openlineage validation failed: path "/": "run" is a required property, path "/": "job" is a required property, path "/": "eventTime" is a required property, path "/": "producer" is a required property, path "/": "schemaURL" is a required property`
+
+Next, send some valid requests:
+
+```shell
+curl -X POST \
+-d "$(cat test-start.json)" \
+-H 'Content-Type: application/json' \
+http://localhost:9880/api/v1/lineage
+```
+
+```shell
+curl -X POST \
+-d "$(cat test-complete.json)" \
+-H 'Content-Type: application/json' \
+http://localhost:9880/api/v1/lineage
+```
+
+After that you should see entities in Marquez (http://localhost:3000/) in the `my-namespace` namespace.
+
+To clean up, run
+```shell
+docker-compose down
+```
+
+## Deployment on Kubernetes
+
+***Section under construction***
+
+## Parser plugin
+
+Openlineage-parser is a Fluentd plugin that verifies if a JSON matches the OpenLineage schema.
+
+### Configuration
```

Comment:
Can we have also single config to disable all validation altogether?

Reply:
Or, to make sure, that only structure of base `RunEvent` is looked at

Reply:
To disable all validation, one should remove the parser from pipeline config. 
To validate `RunEvent` base structure only, all the facets' validations need to be disabled.